src/ResourceManager/RecoveryServices/Commands.RecoveryServices.Backup/RSBackupVaultCmdletBase.cs

```diff
@@ -16,7 +16,8 @@ public abstract class RSBackupVaultCmdletBase : RecoveryServicesBackupCmdletBase
         /// ARM ID of the Recovery Services Vault.
         /// </summary>
         [Parameter(Mandatory = false, HelpMessage = "ARM ID of the Recovery Services Vault.",
-            ValueFromPipeline = true)]
+            ValueFromPipelineByPropertyName = true)]
```

Comment:
@siddharth7 does this scenario currently break users? _i.e._, are users able to write scripts that use this parameter via piping without any exceptions?

Reply:
@siddharth7 resolved offline -- users currently using this piping scenario are broken, so this is not a breaking change. The exception in the build can be suppressed.